EasyPost/Order.cs

```diff
@@ -127,25 +128,12 @@ public static Order Create(Dictionary<string, object> parameters)
         /// </summary>
         /// <param name="id">String representing a Order. Starts with "order_" if passing an id.</param>
         /// <returns>EasyPost.Order instance.</returns>
-        public static Order Retrieve(string id)
+        public static async Task<Order> Retrieve(string id)
         {
             Request request = new Request("orders/{id}");
             request.AddUrlSegment("id", id);
 
-            return request.Execute<Order>();
-        }
-
-        private static Order SendCreate(Dictionary<string, object> parameters)
```

Comment:
Please call the removal of this out in the CHANGELOG.

Reply:
It was internal, so no one ever saw it. Dead code.